snap/README.md

```diff
@@ -290,72 +300,64 @@ JWT_SIGNATURE=`echo -n "$JWT_HEADER.$JWT_PAYLOAD" | openssl dgst -sha256 -binary
 TOKEN=$JWT_HEADER.$JWT_PAYLOAD.$JWT_SIGNATURE
 ```
 
-The resulting JWT token must be included
+4. Once you have the token you can access the API Gateway as follows:
+
+The JWT token must be included
 via an HTTP `Authorization: Bearer <access-token>` header on any REST calls used to access EdgeX services via the API Gateway. 
 
 Example:
 
 ```bash
-$ curl -k -X GET https://localhost:8443/coredata/api/v1/ping? -H "Authorization: Bearer $TOKEN"
-```
-
-Additional users can be added by repeatedly calling the secrets-config command as above. Only one user can however be set at any time when using the snap configure hook, so
-the current user must first be removed by setting the snap configuration settings to an empty string, before setting the values again:
-
-```bash
-$ sudo snap set edgexfoundry env.security-proxy.user=""
-$ sudo snap set edgexfoundry env.security-proxy.public-key=""
-$ sudo snap set edgexfoundry env.security-proxy.user=user02,USER_ID2,ES256
-$ sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
+$ curl -k -X GET https://localhost:8443/core-data/api/v2/ping? -H "Authorization: Bearer $TOKEN"
```

Comment:
Remove the `$` character for consistency with other commands in this section.

Reply:
curl -k -X GET https://localhost:8443/core-data/api/v2/ping? -H "Authorization: Bearer $TOKEN"
{"message":"Bad token; invalid JSON"}

(I think there should be a blank after "Authorization")
curl -k -X GET https://localhost:8443/core-data/api/v2/ping? -H "Authorization : Bearer $TOKEN"
{"message":"Unauthorized"}

Do you have any idea why it is Unauthorized? Thanks!

Reply:
modified.

The reason you get Unauthorized is because of the blank - it should not be added after "Authorization". However, the reason you get "Bad token" is probably because the token is invalid. Did you follow the steps to add a user and create a token?

Reply:
Yes, I added a user and created a token. After changing working directory, now it works. 

I got: `"message":"An invalid response was received from the upstream server"`. Does this mean self-signed TLS  is implemented?